src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/DataSource.scala

```diff
@@ -26,8 +27,12 @@ object DataSource {
   }
 }
 
-class SlickDataSource(val databaseConfig: DatabaseConfig[JdbcDriver])(implicit executionContext: ExecutionContext) extends LazyLogging {
-  private val database = databaseConfig.db
+class SlickDataSource(initialDatabaseConfig: DatabaseConfig[JdbcDriver])(implicit executionContext: ExecutionContext) extends LazyLogging {
+  val dataAccess = new DataAccessComponent(initialDatabaseConfig.driver, initialDatabaseConfig.config.getInt("batchSize"))
+  val databaseConfig = DatabaseConfig.forConfig[JdbcDriver]("", initialDatabaseConfig.config.withValue("db.connectionInitSql", ConfigValueFactory.fromAnyRef(dataAccess.createTempTables)))
```

Comment:
it is in src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DataAccess.scala

I did not want it in actual config because this is really code.